tools/crm_resource.c

```diff
@@ -2031,6 +2036,15 @@ main(int argc, char **argv)
                 params = pcmk__strkey_table(free, free);
                 get_meta_attributes(params, rsc, current, data_set);
 
+            } else if (pcmk__str_eq(options.attr_set_type, XML_TAG_ELEMENT, pcmk__str_casei)) {
+                params = pcmk__strkey_table(free, free);
+
+                // TODO:
+                // - Get element attributes into a hash table, similar to get_meta_attributes above.
+                // - The table maps attribute names to values.
+                // - For example, description maps to "test_description"
+                // - Then I am done.
+            
```

Comment:
Rather than create a hash table, it might be easier just to iterate through the attributes on rsc->xml (see existing loops using pcmk__xe_first_attr() and xmlAttr)